docs/Developers/Newcomers_Guide.rst

```diff
@@ -3,74 +3,83 @@
 Welcome to the Newcomers Guide!
 ===============================
 
-**DO NOT WORK ON ANY ISSUE WITHOUT ASSIGNMENT!** If you do, someone else might
-work on it as well and we might have no choice but reject one of your Pull
-Requests - we hate it if anyone wastes their time. For your own sake, please
-follow this guide. We put a lot of work into this for you!
-
-Everyone in the coala community is expected to follow our
-`Code of Conduct <http://coala.io/coc>`_.
-
-To become part of the coala developers team, there's a few steps you need
-to complete. The newcomer process is as it follows:
-
-You will start as a newcomer, which is kind of a trial. If you complete the
-following tasks, you will become a developer at coala:
-
-  - run coala on a project of yours
-  - merge a ``difficulty/newcomer`` Pull Request
-  - review at least a ``difficulty/newcomer`` Pull Request
-  - merge a ``difficulty/low`` Pull Request
-  - review at least a ``difficulty/low`` or higher Pull Request
-
-When you ran coala on a project, please fill our
-`usability survey <http://coala.io/usability>`_. Once you got your first Pull
-Request merged successfully, fill in our
-`survey form <http://coala.io/newform>`_. With that you can help us making your
-experience better!
-
-Once you have achieved all these, just ask for being promoted on the chat and
-provide links to your reviews and merged Pull Requests. Then, you will be able
-to name yourself a coala developer!
+This is a step by step guide that teaches you the workflow used at coala and
+how to use the tools involved with it.
+Although we did our best to provide as much help as possible, we are always
+there to help at http://coala.io/chat if you feel lost.
+**But please do read the guide before asking us as most questions we get are
+covered in here.**
+
+Before you can start working there are a few things you need to know about
+coala:
+
+  - Everyone in the coala community is expected to follow our
+    `Code of Conduct <http://coala.io/coc>`_.
+  - Do not work on any issue without assignment. Someone else might be working
```

Comment:
but seriously if there's one thing that I want them to know it's this, that's why I put it on top because otherwise we get lots of people without assignment